libmscore/mscore.cpp

```diff
@@ -164,6 +164,7 @@ void Direction::fillComboBox(QComboBox* cb)
       }
 
 static Spatium doubleToSpatium(double d)       { return Spatium(d); }
+static Direction intToDirection(int i)         { return Direction(i); }
```

Comment:
I think we probably don't the functions that use it much or we would see major problems. I just added a converter for TextStyleType https://github.com/musescore/MuseScore/commit/96aa04ecd4131bf8a5b95358f80570329657d93c because the v.value was detected by the scripting tests.